cli/web_worker.rs

```diff
@@ -124,6 +132,22 @@ impl WebWorker {
       ops::errors::init(isolate, &state);
       ops::timers::init(isolate, &state);
       ops::fetch::init(isolate, &state);
+
+      if has_deno_namespace {
+        let op_registry = isolate.op_registry.clone();
+        ops::runtime_compiler::init(isolate, &state);
+        ops::fs::init(isolate, &state);
+        ops::fs_events::init(isolate, &state);
+        ops::plugins::init(isolate, &state, op_registry);
+        ops::net::init(isolate, &state);
+        ops::tls::init(isolate, &state);
+        ops::os::init(isolate, &state);
+        ops::permissions::init(isolate, &state);
```

Comment:
Ops registration in different workers seems ripe for abstraction: maybe `fn register_ops(&mut self)`?